cmd/gpu_plugin/labels.md

```diff
@@ -0,0 +1,52 @@
+# Labels created by GPU Plugin and NFD hook
+
+## Default labels
+
+Following labels are created by default. You may turn numeric labels into extended resources with NFD.
+
+name | type | description|
+-----|------|------|
+|`gpu.intel.com/millicores`| number | node GPU count * 1000. Can be used as a finer grained shared execution fraction.
+|`gpu.intel.com/memory.max`| number | sum of detected [GPU memory amounts](#gpu-memory) in bytes OR environment variable value * GPU count
```

Comment:
While this is copy from earlier text, I'm not happy with "Can be used as a finer grained shared execution fraction", as it implies there being some enforcement, when there's none.  Maybe best just to drop it?

However, I think this should mention something about RM, maybe "RM" column listing items (with 'x') which are relevant only with/for RM, and link/explanation to what that means?